### PR TITLE
Added Examples for Request Header Fields

### LINKED ISSUE #2448: [UI] Add example usage in each of the Performance Test's Request Header fields
#### Current Behavior
The performance test form is not as intuitive as it could be. Users need example values in the fields in order to better use the tool.

Screen Shot 2021-02-23 at 11 42 19 AM

#### Desired Behavior
Show example values in each field.
Request Header (example: {"host":"bookinfo.meshery.io"})

#### Mockups
Screen Shot 2021-02-23 at 11 42 19 AM

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

**Description**

This PR fixes #2448  

**Notes for Reviewers**
The examples added for Performance Test's Request Header Fields are,

- Request Headers: "host":"bookinfo.meshery.io"
- Request Cookies: "yummy_cookie=choco_chip"
- Content Type:  "application/json"
- Response Body: {"method":"post","url":"http://bookinfo.meshery.io/test"}

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
Hey @utkarsh-pro
Could you please review this ? 👀

File: ui/components/MesheryPerformance/index.js [hunk 1]
```diff
@@ -888,7 +888,7 @@ class MesheryPerformanceComponent extends React.Component {
                         <TextField
                           id="cookies"
                           name="cookies"
-                          label="Request Cookies"
+                          label='Request Cookies e.g. "yummy_cookie=choco_chip"'
```
Comment (utkarsh-pro) [inline ui/components/MesheryPerformance/index.js:891]: @adithyaakrishna can you please make this change?

```diff
- label='Request Cookies e.g. "yummy_cookie=choco_chip"'
+ label='Request Cookies e.g. {"yummy_cookie": "choco_chip"}'
```

File: ui/components/MesheryPerformance/index.js [hunk 2]
```diff
@@ -901,7 +901,7 @@ class MesheryPerformanceComponent extends React.Component {
                         <TextField
                           id="contentType"
                           name="contentType"
-                          label="Content Type"
+                          label='Content Type e.g. "application/json"'
```
Comment (utkarsh-pro) [inline ui/components/MesheryPerformance/index.js:904]: @adithyaakrishna please remove the double quotes, ie
```diff
- label='Content Type e.g. "application/json"'
+ label='Content Type e.g. application/json'
```

The text is already stringified, so the double quotes would be interpreted literally.

File: ui/components/MesheryPerformance/index.js [hunk 3]
```diff
@@ -914,7 +914,7 @@ class MesheryPerformanceComponent extends React.Component {
                         <TextField
                           id="cookies"
                           name="cookies"
```
Comment (utkarsh-pro) [inline ui/components/MesheryPerformance/index.js:916]: @adithyaakrishna please consider changing this and id to "request body"

Comment (adithyaakrishna) [reply ui/components/MesheryPerformance/index.js:916]: Okay Sure

File: ui/components/MesheryPerformance/index.js (hunk 2 above)
Comment (adithyaakrishna) [reply ui/components/MesheryPerformance/index.js:904]: Okay

**Review by utkarsh-pro — APPROVED** [review]
LGTM. @adithyaakrishna I think there is some DCO issue, please check it out.

**Comment by adithyaakrishna** [comment]
I've signed the DCO 👀

**Review by utkarsh-pro — DISMISSED** [review]

**Comment by utkarsh-pro** [comment]
Great work @adithyaakrishna! :smile:

**Review by utkarsh-pro — APPROVED** [review]

**Pull request merged**